.github/workflows/unit_tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        python-version: [ 3.7, 3.8, 3.9, "3.10" ]
+        python-version: [ 3.8, 3.9, "3.10" ]
```

Comment:
Future feature: testing with 3.11, 3.12, and maybe 3.13 if it's out by the time we implement this